main.css

```diff
@@ -1,3 +1,37 @@
 body {
-  background-color: beige;
+  background-color: black;
+}
+
```

Comment:
write comment showing this is for the footer section

Reply:
Done!